clarifai/runners/models/model_upload.py

```diff
@@ -258,6 +258,18 @@ def upload_model_version(self, download_checkpoints):
     file_path = f"{self.folder}.tar.gz"
     logger.info(f"Will tar it into file: {file_path}")
 
+    model_type_id = self.config.get('model').get('model_type_id')
+
+    if not download_checkpoints and not os.path.exists(self.checkpoint_path) and (
```

Comment:
instead of just exists(checkpoint_path) should it use the checkpoint loader validate function to make sure it's really a properly loaded checkpoint directory?

Reply:
yeah, that would be better just commit the update